tests/test_visitors/test_ast/test_conditions/test_useless_len_call.py

```diff
@@ -5,16 +5,16 @@
 )
 from wemake_python_styleguide.visitors.ast.conditions import IfStatementVisitor
 
-correct_call1 = 'print(len([]))'
+correct_call = 'print(len([]))'
```

Comment:
I have a question about this case.

What do you think:
1. Should we allow `[a-z]\d+$` pattern? (1 and other numbers can always be located at the end of the variable name)
2. Should we stick to this implementation?

I, personally, like the first one more.

Reply:
Should not we check only `'l1+$` because other cases correct by default?

Reply:
Yes, you are right.

Reply:
Oh, one more: `O0`

Reply:
I suggest to change WPS cases like correct_call1 to correct_call_1 and do not allow any `(l1|O0)+$`. It seems strange to me allow smth in one case and disallow in other

Reply:
> It seems strange to me allow smth in one case and disallow in other

I agree, we value consistency. 
But, `correct_call_1` is not allowed due to our underscore policies. 

I would still allow `l1$` because it feels like a popular pattern people should be aware about.

Reply:
Also see UnderscoredNumberNameViolation. But still think that unreadable is still unreadable even in the end of name

Reply:
@AlwxSin what do you think? You have edited this rule recently, maybe you have some clever way to solve it.